eng/tools/spec-gen-sdk-runner/src/commands.ts

```diff
@@ -130,6 +132,21 @@ export async function generateSdkForSpecPr(): Promise<number> {
       // Read the execution report to aggreate the generation results
       executionReport = getExecutionReport(commandInput);
       currentExecutionResult = executionReport.executionResult;
+
+      if (executionReport.stagedArtifactsFolder) {
+        stagedArtifactsFolder = executionReport.stagedArtifactsFolder;
+      }
+      const apiViewArtifactRelPath = path.relative(executionReport.stagedArtifactsFolder, executionReport.sdkApiViewArtifactFolder);
```

Comment:
`executionReport.sdkApiViewArtifactFolder)` is an optional property, we will need to do null check.